### PR TITLE
Fixed build_done #2. Fixed logs handling and multilib

### PR DESCRIPTION
**Pull request by Korulag**

### CODE REVIEW

File: alws/crud/build_node.py [hunk 1]
```diff
@@ -159,8 +159,8 @@ def get_repo(repo_arch, is_debug):
                     await pulp_client.modify_repository(
                         repo.pulp_href, add=hrefs)
                 except Exception as e:
-                    logging.error('Cannot add RPM packages to the repository: %s',
-                                  str(e))
+                    logging.exception('Cannot add RPM packages '
+                                      'to the repository: %s', e)
```
Comment (anfimovdm) [inline alws/crud/build_node.py:163]: there should be repo name instead of exception

File: alws/crud/build_node.py [hunk 2]
```diff
@@ -175,8 +175,8 @@ def get_repo(repo_arch, is_debug):
         )
 
     if module_index:
-        results = await asyncio.gather(*[__get_rpm_package_info(
-            pulp_client, rpm.href) for rpm in rpms])
+        results = await asyncio.gather((__get_rpm_package_info(
+            pulp_client, rpm.href) for rpm in rpms))
```
Comment (anfimovdm) [inline alws/crud/build_node.py:179]: not sure what `gather` can unpack generator

File: alws/utils/pulp_client.py [hunk 3]
```diff
@@ -513,25 +523,16 @@ async def request(
...
-
-async def create_entity(pulp_client: PulpClient, artifact):
-    if artifact.type == 'rpm':
-        entity_href = await pulp_client.create_rpm_package(
-            artifact.name, artifact.href)
-    else:
-        entity_href = await pulp_client.create_file(
-            artifact.name, artifact.href)
-    return entity_href, artifact
+            async with RetryClient() as client:
```
Comment (anfimovdm) [inline alws/utils/pulp_client.py:526]: i'm think we should define statuses to retry

**Review by anfimovdm — CHANGES_REQUESTED** [review]

File: alws/crud/build_node.py (hunk 2 above)
Comment (Korulag) [reply alws/crud/build_node.py:179]: It actually unpacks it, tested locally

**Pull request merged**